modules/nw-sctp-about.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-about_{context}"]
+= Support for Stream Control Transmission Protocol (SCTP) on {product-title}
+
+As a cluster administrator, you can enable SCTP on the hosts in the cluster.
+On {op-system-first} the SCTP module is blacklisted by default.
```

Comment:
On {op-system-first},

Can you add a little more context about why it's blacklisted or what that means for the user?

Reply:
@kalexand-rh, it's due to an oversight. Perhaps _blacklisted_ is too harsh. (Although that's the technical term, as the kernel cannot load the SCTP module until it is removed from the _blacklist_.) Not sure how to diplomatically say "sorry, our bad."

Could say

> On {op-system-first}, the SCTP module is disabled by default.